src/components/ContactUs/OnDemand/formik-hooks/useFormikStep3.ts

```diff
@@ -116,10 +116,14 @@ const useFormikStep3 = ({
 
       const topicValues = () => {
         if (otherTopicCheckbox) {
-          return topic.concat(',', otherTopic)
+          if (topic) {
+            return topic.concat(',', otherTopic)
+          } else {
+            return otherTopic
+          }
+        } else {
+          return topic
```

Comment:
```suggestion
    const topicValues = () => {
       if (otherTopicCheckbox && topic) {
           return topic.concat(',', otherTopic);
        }
       return topic || otherTopic;
    };
```
gini lebih enak gak sii

Reply:
iya jugaa